src/js/core/player.js

```diff
@@ -727,12 +727,57 @@ function name() {
     const country = playerNames.countries.find(row => row[1] >= cRand)[0];
 
     // First name
-    const fnRand = random.uniform(0, playerNames.first[country][playerNames.first[country].length - 1][1]);
-    const firstName = playerNames.first[country].find(row => row[1] >= fnRand)[0];
+    let fnRand = random.uniform(0, playerNames.first[country][playerNames.first[country].length - 1][1]);
+    let firstName = playerNames.first[country].find(row => row[1] >= fnRand)[0];
+
+    // Middle Name, randomly
```

Comment:
So from my view, a *simple name* is one which has only one "part". It's a subset of the known names and excludes anything with spaces, hyphens, or periods. That way I don't draw a name like "{John} {J.R.} {Smith}" for first-middle-last. The intention is to reroll the name until I get a two usable "simple names" and build the name from those components.